safe/gui/tools/options_dialog.py

```diff
@@ -412,7 +418,7 @@ def on_toolNorthArrowPath_clicked(self):
 
     # noinspection PyPep8Naming
     @pyqtSignature('')  # prevents actions being handled twice
-    def on_toolOrganisationLogoPath_clicked(self):
+    def on_open_organisation_logo_path_button_clicked(self):
         """Auto-connect slot activated when logo file tool button is clicked.
```

Comment:
Can we remove these autoconnects while you are working on this file?

Reply:
Ok, I will. Thanks for the review.